src/Ledger/Conway/Specification/Epoch.lagda

```diff
@@ -535,7 +557,29 @@ its results by carrying out each of the following tasks.
 \end{figure*}
 
 \begin{NoConway}
+The \AgdaFunction{calculatePoolDistr} produces a new pool distribution from the
+delegation map and stake allocation of the previous epoch.
+
 \begin{figure*}[ht]
+\begin{code}
+opaque
+  calculatePoolDistr : Snapshot → PoolDelegatedStake
+  calculatePoolDistr ss =
```

Comment:
I wonder if we should keep or change the name of this function since it is not calculating the PoolDistr as in the Shelley spec, but something else.

Reply:
The name makes easy to find the corresponding spot in the Agda spec. We could still change it to something like
```
\AgdaFunction{calculatePoolDelegatedStake} performs the computation of
\AgdaFunction{calculatePoolDistr} in the Shelley spec, without normalizing the
stakes to be between 0 and 1.
\begin{code}
calculatePoolDelegatedStake : Snapshot → PoolDelegatedStake
...
```